lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -217,13 +218,19 @@ IncrementalJIT::getSymbolAddressWithoutMangling(llvm::StringRef Name,
   if (auto Sym = getInjectedSymbols(Name))
     return Sym;
 
+  const std::string SymName = Name;
```

Comment:
So you're the one!
It's a length and a pointer no null termination garauntee.
llvm::StringRef("chopmeup").substr(0,4).data() -> "chopmeup"
llvm::StringRef("chopmeup").substr(0,4).str().c_str() -> "chop"